### PR TITLE
add filter by option

### PR DESCRIPTION
**Pull request by omaretna**
Make sure to read [the contributing documentation](https://github.com/apidoc/apidoc/blob/master/CONTRIBUTING.md).

IMPORTANT: Base your PR off the 'dev' branch and target that branch for merging.

This PR is related to https://github.com/apidoc/apidoc-core/pull/91

### CODE REVIEW

File: bin/apidoc [hunk 1]
```diff
@@ -55,6 +55,8 @@ var argv = cmd
 
     .option('--encoding <encoding>', 'Set the encoding of the source code. [utf8].', 'utf8')
 
+    .option('--filter-by <tag-filter:value>', 'Filter doc by tag')
```
Comment (NicolasCARPi) [inline bin/apidoc]: Please use `documentation` in full ;)

Comment (omaretna) [reply bin/apidoc]: @NicolasCARPi I don't understand what you mean by that :)

Comment (NicolasCARPi) [reply bin/apidoc]: instead of "doc" write "documentation"

Comment (omaretna) [reply bin/apidoc]: ah hh oké :)

Comment (omaretna) [reply bin/apidoc]: Done

**Comment by danielskeeled** [comment]
This is part of the documentation, but there's two issues:
1 - In the documentation the delimiter is supposed to be: "``:``", but in the code it's splitting by: "``=``";
2 - Even though it's documented it's not possible to use since the apidoc-core version listed in the project dependencies (``"apidoc-core": "~0.8.3",``) is only being updated on patch changes and this was a minor change.

**Comment by NicolasCARPi** [comment]
@danielskeeled 

1. Good catch, I fixed it.
2. In package.json of apidoc the apidoc-core version is ^0.11.0, so I don't understand your point.

**Comment by omaretna** [comment]
@NicolasCARPi 
> 2. In package.json of apidoc the apidoc-core version is ^0.11.0, so I don't understand your point.
You need to release a new version of apidoc the last release don't contain ^0.11.0.

Thanks.

**Comment by NicolasCARPi** [comment]
> You need to release a new version of apidoc the last release don't contain ^0.11.0.

Oh I see now! I was thinking that master branch was latest published version but it is not!

**Comment by NicolasCARPi** [comment]
@rottmann Please `npm publish` 0.19.1 ;)

**Comment by rottmann** [comment]
@NicolasCARPi Done!

**Comment by NicolasCARPi** [comment]
@rottmann It's still showing as 0.19.0 on https://www.npmjs.com/package/apidoc, but maybe it takes more time to update. I was under the impression that it was almost immediate... :thinking:

**Comment by rottmann** [comment]
@NicolasCARPi Ups, published it before on my private registry. Now it is on npm-Registry.

**Pull request merged**